packages/assets-controllers/src/TokensController.test.ts

```diff
@@ -1545,7 +1550,7 @@ describe('TokensController', () => {
         .spyOn(tokensController as any, 'getNetworkClientById')
         .mockReturnValue({
           configuration: { chainId: '0x5' },
-          provider: sinon.stub(),
+          provider,
```

Comment:
Following my comment above, you can also create a new FakeProvider in this test and use it for this property as well.